mlkem/native/x86_64/src/arith_native_x86_64.h

```diff
@@ -37,7 +37,11 @@ void reduce_avx2(__m256i *r, const __m256i *qdata);
 
 #define basemul_avx2 MLKEM_NAMESPACE(basemul_avx2)
 void basemul_avx2(__m256i *r, const __m256i *a, const __m256i *b,
-                  const __m256i *qdata);
+                  const __m256i *qdata, const __m256i *bcache);
+
+#define poly_mulcache_compute_avx2 \
+  MLKEM_NAMESPACE(poly_mulcache_compute_avx2)
```

Comment:
@dkostic can you run `format` to appease the linter? You just need to install nix and run 
```
nix develop --experimental-features 'nix-command flakes'
```
Then `format` and `lint` are in your PATH and configured in the same way as in CI.

Reply:
I tried to do that but I get the following error:
```
➜  mlkem-native sudo nix develop --experimental-features 'nix-command flakes'

error: flake 'path:/home/ubuntu/workspace/mlkem-native' does not provide attribute 'devShells.x86_64-linux.devShell.x86_64-linux', 'packages.x86_64-linux.devShell.x86_64-linux', 'legacyPackages.x86_64-linux.devShell.x86_64-linux', 'devShell.x86_64-linux' or 'defaultPackage.x86_64-linux'
```

Reply:
@potsrevennil does that ring a bell?

Reply:
@dkostic Which version of nix are you running?

Reply:
I'd also suspect it's due to the nix version, maybe we should also add to the document about nix version > 2.18 ?

Reply:
I was wrong, I did some tests with nix 2.17.2 docker image, and everything works as usual. 
Maybe it's not related to nix version, or maybe we could support nix version > 2.17 (or even older).

Reply:
Thanks @potsrevennil !

@dkostic how did you install nix?

Reply:
`sudo apt-get install nix`